internal/service/contentprovider/moduleconfig.go

```diff
@@ -68,6 +70,12 @@ func (s *ModuleConfigProvider) validateArgs(args types.KeyValueArgs) error {
 	return nil
 }
 
+type Manager struct {
+	Name                    string `yaml:"name" comment:"required, the name of the manager"`
+	Namespace               string `yaml:"namespace" comment:"optional, the path to the manager"`
```

Comment:
Honestly I don't think we need to do this because with the current implementation for both cases (no namespace or just empty namespace), we don't have the namespace field in the ModuleTemplate

Reply:
Also okay 👍🏻